include/ui/theme.hpp

```diff
@@ -117,6 +118,17 @@ class Theme : public QQmlPropertyMap {
   static Theme* _theme;
 
   /**
+   * Attempts to copy a theme from the resource directory (located in the
+   * theme/ directory relative to the executable/working directory) into the
+   * given destination directory.
+   *
+   * \param name The name of the theme to copy.
+   * \param destination the destination directory, to copy the theme into.
+   * \return true if theme could be copied into the destination directory.
+   */
+  static bool _copyTheme(const QString& name, QDir destination);
```

Comment:
Actually, there is a reason for that: destination is changed during `_copyTheme`, due to a call to `cd()`. If destination it's a const reference, `cd()` can't be called and if its just a reference, the calling function won't work correctly. So i figured, call by value would be the best in this case.